### PR TITLE
Extract common parts from Parameter Store and Secrets Manager integrations

### PR DESCRIPTION
**Pull request by maciejwalkowiak**

### CODE REVIEW

File: v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoader.java [hunk 1]
```diff
@@ -43,11 +37,10 @@
...
 public class ParameterStoreConfigDataLoader implements ConfigDataLoader<ParameterStoreConfigDataResource> {
 
-	private final DeferredLogFactory logFactory;
-
 	public ParameterStoreConfigDataLoader(DeferredLogFactory logFactory) {
-		this.logFactory = logFactory;
-		reconfigureLoggers(logFactory);
+		BootstrapLoggingHelper.reconfigureLoggers(logFactory,
+				"io.awspring.cloud.v3.paramstore.ParameterStorePropertySource",
+				"io.awspring.cloud.v3.paramstore.ParameterStorePropertySource");
```
Comment (MatejNedic) [inline v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoader.java:43]: there should be one entry of ParameterStorePropertySource.

File: v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/config/secretsmanager/SecretsManagerConfigDataLoader.java [hunk 2]
```diff
@@ -44,11 +38,10 @@
...
 public class SecretsManagerConfigDataLoader implements ConfigDataLoader<SecretsManagerConfigDataResource> {
 
-	private final DeferredLogFactory logFactory;
-
 	public SecretsManagerConfigDataLoader(DeferredLogFactory logFactory) {
-		this.logFactory = logFactory;
-		reconfigureLoggers(logFactory);
+		BootstrapLoggingHelper.reconfigureLoggers(logFactory,
+				"io.awspring.cloud.v3.secretsmanager.SecretsManagerPropertySource",
+				"io.awspring.cloud.v3.secretsmanager.SecretsManagerPropertySource");
```
Comment (MatejNedic) [inline v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/config/secretsmanager/SecretsManagerConfigDataLoader.java:44]: Same

**Review by MatejNedic — APPROVED** [review]
Looks good to me. Just one small thing with loggers.

File: v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoader.java (hunk 1 above)
Comment (maciejwalkowiak) [reply v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoader.java]: Ah, it should be actually different class. thx good spot.

**Pull request merged**